src/threading/pss_common.h

```diff
@@ -1,112 +0,0 @@
-/*
```

Comment:
`cthread` is now using just a simple, serial `std::sort` implementation. Which is not ideal, but then we were having all those nvcc errors trying to compile the PSS code.

We should definitely consider reintroducing a thread-parallel sort, but I think we're best off writing our own, in order to avoid the nvcc issue, and also to have one that is independent of OpenMP.